src/Routes.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Home from './pages/HomeComponent';
+import CustomNavbar from './components/NavbarComponent';
+import NotFound from './pages/NotFound';
+import Login from './pages/LoginComponent';
+import Signup from './pages/Signup';
+import ConfirmEmail from './pages/ConfirmRegistration';
+import EventDetails from './pages/SingleEventComponent';
+import Events from './pages/EventsComponent';
+
+class Routes extends Component {
+  render(){
+    return(
+      
+        <BrowserRouter>
+          <div>
+            <CustomNavbar />
+              <Switch>
```

Comment:
Fix this indentation. It appears as though *Switch* is a child component of *CustomNavbar* instead of sibling